app/src/renderer/components/transactions/LiTransaction.vue

```diff
@@ -39,8 +39,8 @@ export default {
       default: null
     },
     time: {
-      type: Number,
-      default: null
+      type: String,
+      default: null // TODO: fails with required: true
```

Comment:
why?

Reply:
idk

Reply:
that is not really a satisfactory answer... 

should it be required?